src/include/simeng/SpecialFileDirGen.hh

```diff
@@ -7,6 +7,9 @@
 #include "simeng/version.hh"
 
 namespace simeng {
+
+/** Path to the root of the SimEng special files directory. */
+static const std::string specialFilesDir_ = SIMENG_BUILD_DIR "/specialFiles";
```

Comment:
If this is now static, do we need the same private member variable in the syscall handler?

Reply:
Removed